src/common/transformations/include/transformations/common_optimizations/TotalSequenceLengthPattern.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "openvino/op/parameter.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "openvino/cc/pass/itt.hpp"
+#include "transformations/utils/utils.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API TotalSequenceLengthPattern;
```

Comment:
export via `TRANSFORMATIONS_API ` is not required

Reply:
done